package/network/services/hostapd/Makefile

```diff
@@ -133,6 +133,10 @@ ifneq ($(LOCAL_TYPE),hostapd)
   ifdef CONFIG_WPA_SUPPLICANT_NO_TIMESTAMP_CHECK
     TARGET_CFLAGS += -DNO_TIMESTAMP_CHECK
   endif
+  ifeq ($(LOCAL_VARIANT),mini)
+    TARGET_CFLAGS += -flto -Os
```

Comment:
Remove -Os, it's already in CFLAGS by default.

Reply:
Done.